schema/deploy/tables/ciip_user_organisation_001_bceid.sql

```diff
@@ -0,0 +1,9 @@
+-- Deploy ggircs-portal:tables/ciip_user_organisation_001_bceid to pg
+-- requires: tables/ciip_user_organisation
+
+begin;
+
+alter table ggircs_portal.ciip_user_organisation add column bceid_business_name varchar(1000);
```

Comment:
There are scenarios (company merge, business bceid org reset) where an existing BCeID can get a new business name. We'd probably need to invalidate something if that was the case? Or figure how to handle that case with the business area.